### PR TITLE
Bug/multiple menu

### PR DESCRIPTION
**Pull request by ZhengHe-MD**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,6 +59,7 @@
   },
   "dependencies": {
     "font-awesome": "4.6.3",
-    "jquery": "3.1.0"
+    "jquery": "3.1.0",
+    "uuid": "2.0.2"
```
Comment (jimexist) [inline package.json]: i think there should be a built-in uuid library by node? no?

File: src/MDDropDownMenu.jsx [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 import $ from 'jquery';
+import uuid from 'uuid';
 
 class MDDropDownMenu extends Component {
 
   constructor(props) {
     super(props);
     this.state = {
       defaultOption: this.props.defaultOption,
+      id: uuid.v1(),
```
Comment (jimexist) [inline src/MDDropDownMenu.jsx]: prefer to use `v4`

**Comment by jimexist** [comment]
LGTM

File: package.json (hunk 1 above)
Comment (ZhengHe-MD) [inline package.json]: thx for pointing that out. Node has 'node-uuid'

File: src/MDDropDownMenu.jsx (hunk 2 above)
Comment (ZhengHe-MD) [inline src/MDDropDownMenu.jsx]: why?

**Pull request merged**